Take a Break/naman.py

```diff
@@ -0,0 +1,11 @@
+import time
+import webbrowser
+i = 0
+x = 3
+print("The program started on " + time.ctime())
+while(i < x):
+  time.sleep(1800)
+  print(str(i + 1) + "break is taking place on " + time.ctime())
+  webbrowser.open("https://www.youtube.com/")
+  time.sleep(1200)
```

Comment:
20 minutes break after every 30 minutes is too much.
And can you please notify the user after completion of break time, or close the YouTube instance which was opened by this script.

Reply:
I added a new branch with solving the problem and incorporating the required changes.